src/pages/index.tsx

```diff
@@ -67,7 +72,15 @@ const HomepageSearch: React.FunctionComponent = () => {
     );
 };
 
-const Home: NextPage = () => {
+interface HomeProps {
+    recommendedTopics: MetaInfo[];
```

Comment:
Can we just have this be `Tag[]` instead? We're only using this data as a `Tag`, so it might be more clear if we just map the `MetaInfo`s to `Tag`s in `getStaticProps` just so we're dealing with the same interface up until we generate the `TopicCard`s.